assets/src/edit-story/components/checklist/karma/checklist.karma.js

```diff
@@ -179,19 +208,113 @@ describe('Checklist integration', () => {
       expect(fixture.editor.checklist.designPanel).toBeDefined();
       expect(fixture.editor.checklist.accessibilityPanel).toBeNull();
 
-      // TODO #8085 - a11y section not available in blank page state, no issues present
+      // add accessibility section
+      await addAccessibilityIssue();
       // tab to accessibility section
-      // while (
-      //   fixture.editor.checklist.accessibilityTab !== document.activeElement
-      // ) {
-      //   // eslint-disable-next-line no-await-in-loop
-      //   await fixture.events.keyboard.press('tab');
-      // }
-
-      // await fixture.events.keyboard.press('Enter');
-      // expect(fixture.editor.checklist.priorityPanel).toBeNull();
-      // expect(fixture.editor.checklist.designPanel).toBeNull();
-      // expect(fixture.editor.checklist.accessibilityPanel).toBeDefined();
+      while (
```

Comment:
we might want to introduce some count break here here so it doesn't hang in the `while`

Reply:
i'll do that in my PR that claudio is testing.